gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -172,6 +172,13 @@
               </div>
             </div>
             <div class="col-md-6">
+              <div class="form-group" ng-class="{'has-error': settingsEditor.UnixPerms.$invalid && settingsEditor.UnixPerms.$dirty}">
```

Comment:
`ng-if="tmpGUI.address.startswith('unix://')"` ?

Reply:
Ya, thanks. In this case, though, only '/' passes address validation.